### ISSUE TITLE
Escaping forward slash

### ISSUE THREAD

**Issue by kszafran** [eiiches/jackson-jq#51]
I accidentally stumbled upon another incompatibility with jq (jq-1.6):
```
$ jq -n '"\/"'
"/"
```

with jackson-jq (0.0.11):
```
...
Caused by: java.lang.IllegalStateException: Unexpected escape char: /
	at net.thisptr.jackson.jq.internal.javacc.JsonQueryParser.StringInterpolation(JsonQueryParser.java:1428)
	at net.thisptr.jackson.jq.internal.javacc.JsonQueryParser.PrimaryExpression(JsonQueryParser.java:512)
	at net.thisptr.jackson.jq.internal.javacc.JsonQueryParser.BinaryOperatorExpression(JsonQueryParser.java:271)
	at net.thisptr.jackson.jq.internal.javacc.JsonQueryParser.TupleExpression(JsonQueryParser.java:244)
	at net.thisptr.jackson.jq.internal.javacc.JsonQueryParser.JsonQuery(JsonQueryParser.java:171)
	at net.thisptr.jackson.jq.internal.javacc.JsonQueryParser.Start(JsonQueryParser.java:162)
	at net.thisptr.jackson.jq.internal.javacc.JsonQueryParser.compile(JsonQueryParser.java:23)
	at net.thisptr.jackson.jq.JsonQuery.compile(JsonQuery.java:38)
	... 3 more
```

It seems that jq allows escaping forward slashes, and JSON does too (https://stackoverflow.com/questions/1580647/json-why-are-forward-slashes-escaped). Perhaps it was carried over from JavaScript, where escaping `/` might be required when you want to write a regex like `/\//`...

**Comment by eiiches** [comment]
Thanks for reporting this. I didn't know `\/` is valid in a JSON string...
I'll probably work on this issue sometime in the next week :)

**Comment by eiiches** [comment]
Fix landed in 0.0.12 and 1.0.0-preview.20191208
